backend/control/control.go

```diff
@@ -169,13 +120,130 @@ func TriggerScan(postgresDb *gorm.DB) error {
 		return err
 	}
 	if sr.Status == "FAILED" {
-		return fmt.Errorf("starting cartography job failed")
+		return fmt.Errorf("Scan Status: Failed")
+	}
+	return nil
+}
+
+func CartographyExecutionLoop(postgresDb *gorm.DB, driver neo4j.Driver, ruleDataList []models.RuleData,
+	rulesToExecute []types.Rule) {
+	for {
+		accountReadyToBeScanned, err := GetAccountReadyToBeScanned(postgresDb)
```

Comment:
convention in golang is for local variable names to be kept short, so better to call this something like `account`